build

```diff
@@ -21,6 +21,12 @@ function makeCachebuster(template, output) {
 		try {
 			const filename = c.replace('//play.pokemonshowdown.com/', '../../play.pokemonshowdown.com/');
 			const fstr = fs.readFileSync(path.resolve(rootDir, filename), {encoding: 'utf8'});
+			if (fstr.indexOf('.template.html') >= 0) {
```

Comment:
You don't need to replace every `fstr`, only `indexContents`.

Reply:
...In hindsight, you do need to replace every `fstr`.

But you still need to also replace every `indexContents`, unless it already does the thing.

Reply:
...which I guess they don't. Nevermind, then.